docs/distribution/README.md

```diff
@@ -0,0 +1,923 @@
+        "digest": "sha256:333cc44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b785c333c",
+        "size": 1906
+      },
+      "manifests": [
+        {
+          "mediaType": "application/vnd.oci.image.manifest.v1+json",
+          "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+          "size": 7023,
+          "platform": {
+            "architecture": "ppc64le",
+            "os": "linux"
+          }
+        },
+        {
+          "mediaType": "application/vnd.oci.image.manifest.v1+json",
+          "digest": "sha256:333mbbf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cb333m",
+          "size": 7023,
+          "config_mediaType": "application/vnd.cncf.notary.config.v2+jwt"
+        }
+      ]
+    }
+    ```
+
+5. **manifest digest for the acme-rockets signature** `sha256:333mc0c33ebc4a74a0a554c86ac2b28ddf3454a5ad9cf90ea8cea9f9e75c333m`
+
+    ```json
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.manifest.v2+json",
+      "config": {
+        "mediaType": "application/vnd.cncf.notary.config.v2+jwt",
+        "digest": "sha256:333cc44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b785c333c",
+        "size": 1906
+      },
+      "layers": []
+    }
+    ```
+
+    The `manifest.config` contains the following signature information:
+
+    ```json
+    {
+        "signed": {
+            "mediaType": "application/vnd.oci.image.manifest.v2+json",
+            "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+            "size": 528,
+            "references": [
+                "registry.acme-rockets.com/net-monitor:v1"
+            ],
+            "exp": 1627555319,
+            "nbf": 1596019319,
+            "iat": 1596019319
+        },
+        "signature": {
+            "typ": "x509",
+            "sig": "UFqN24K2fLj...",
+            "alg": "RS256",
+            "x5c": [
+                "MIIDszCCApugAwIBAgIUL1anEU/..."
+            ]
+        }
+    }
+    ```
+
+**Pros with this approach:**
+
+* Conforms to norms of indexes as collections of manifests with no config data
+
+**Cons with this approach:**
+
+* An additional indirection between the original artifact being signed and the individual signatures.
+* An `index.config.mediaType` is still required to identify the type of index being something other than a multi-arch index.
+
+The implied benefit is the signature is moved from the `index.config` to a layer within a manifest. However, config objects are [oci descriptors][oci-descriptor] pointing to blobs. Whether the signature is stored within a config or a layer is little difference. Since we'll need an `index.config.mediaType` to differentiate a signature index from a multi-arch index, are we justifying additional round trips to get a list of signatures?
+
+## Linking Signatures to Artifacts
+
+A signature is only interesting if it's linked to the object it's signing. The question is how.
```

Comment:
I personally prefer not to add new APIs to distribution-spec for this purpose.

Reply:
To support Notary v1 server, we need to stand up an entire new service. With all the downsides of inconsistent data across the two services (like a tag is different with/without docker_content_trust enabled)
So, while we'd all like to get Notary v2 for free, we don't see a means without:
- hacking something into the existing APIs
- sacrificing the requirements

The latest approaches we're pursuing is adding new APIs that would work across any artifact type, including signatures. 
There is a get-signatures API in the current thinking, but I'm hoping this will become some generic "find-my-dependencies" api in a subsequent iteration.

Can you think of a proposal that meets the requirements without adding new APIs?

Reply:
The whole point of this prototype is to explore options. I'm resolving this one for clarity of status, but we're very interested in other options. 

We are hopeful these additions will serve other artifact scenarios, and not be limited to signatures.